### ISSUE TITLE
Dependency warning when using React 16

### ISSUE THREAD

**Issue by einarolafs** [Semantic-Org/Semantic-UI-React#2192]
### Steps
running `npm install semantic-ui-react` with React v.16.0.0 installed.

### Expected Result
No Warnings

### Actual Result
```
npm WARN semantic-ui-react@0.74.2 requires a peer of react@>=0.14.0 <= 15 but none was installed.
npm WARN semantic-ui-react@0.74.2 requires a peer of react-dom@>=0.14.0 <= 15 but none was installed.
```

### Version
0.74.2

When installing semantic-ui-react with NPM, I am warned that I need to have a version of React less than 16. 

Is version v16 of React not supported?

**Comment by einarolafs** [comment]
I might have rust too quickly to write this issue, so just to be clear. I have just tried to implement some semantic components and it seems to work fine, despite these warnings.

**Comment by layershifter** [comment]
Current version is compatible with React 16 except the warning with `peerDependencies`, all other was fixed in #2131 and will be released soon.
